processing/src/main/java/io/druid/query/groupby/GroupByQueryQueryToolChest.java

```diff
@@ -374,6 +370,18 @@ public String apply(DimensionSpec input)
   }
 
   @Override
+  public QueryRunner<Row> annotateDistributionTarget(QueryRunner<Row> runner)
+  {
+    return (query, responseContext) -> {
```

Comment:
Added a default implementation to QueryToolChest. I think only JoinQueryQueryToolChest which will be added in a follow-up pr will have different implementation.